ui/jobs/jobs.ts

```diff
@@ -396,41 +482,49 @@ class Bars {
     return textDiv;
   }
 
-  addProcBox(options) {
+  addProcBox(options: {
+    id?: string;
+    fgColor?: string;
+    threshold?: number;
+    scale?: number;
+  }): TimerBox {
     const id = this.job.toLowerCase() + '-procs';
 
     let container = document.getElementById(id);
     if (!container) {
       container = document.createElement('div');
       container.id = id;
-      document.getElementById('bars').appendChild(container);
+      document.getElementById('bars')?.appendChild(container);
       container.classList.add('proc-box');
     }
 
-    const timerBox = document.createElement('timer-box');
+    const timerBox = document.createElement('timer-box') as unknown as TimerBox;
```

Comment:
`as unknown`?

Reply:
＞︿＜ Oh, this was written before the TypeScript-ization of `resources/timerbox.js`, and I forgot to delete the `unknown`-cast